power bi sample/readme.md

```diff
@@ -0,0 +1,73 @@
+# Dynamics 365 Fraud Protection Event Tracing Power BI Sample Report
```

Comment:
Regarding the Zip file, how was it generated?  How do we maintain it, if needed?

Reply:
The zip file is a solution which you can package a bunch of things in (eg. CDS entities, apps, etc.). I am writing internal docs for our team to create and package new solutions in the future.